src/js/GaCesium.js

```diff
@@ -178,3 +181,93 @@ var GaCesium = function(map, gaPermalink, gaLayers, gaGlobalOptions, $q) {
 
 };
 
+var SSECorrector = function() {
+  var params = {};
+  location.search.substr(1).split('&').reduce(function(previous, current) {
+    var splitted = current.split('=');
+    if (splitted.length === 2) {
+      params[splitted[0]] = splitted[1];
+    } else {
+      params[splitted[0]] = true;
+    }
+  });
```

Comment:
Temporary for a long time :) . Yes I'll prefer to use the gaPermalink, You can pass the `params` object in the parameter of the constructor.